tests/unit_tests/test_scap_result_parser.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from oscap_report.scap_results_parser.data_structures import Report
+from oscap_report.scap_results_parser.scap_results_parser import \
+    SCAPResultsParser
+
+from ..constants import PATH_TO_ARF, PATH_TO_EMPTY_XML_FILE, PATH_TO_XCCDF
+
+
+def get_parser(file_path):
+    xml_data = None
+    with open(file_path, "r", encoding="utf-8") as xml_report:
+        xml_data = xml_report.read().encode()
+    return SCAPResultsParser(xml_data)
+
+
+@pytest.mark.parametrize("file_path, result", [
+    (PATH_TO_ARF, True),
+    (PATH_TO_XCCDF, False),
+    (PATH_TO_EMPTY_XML_FILE, False),
+])
+def test_validation(file_path, result):
+    parser = get_parser(file_path)
+    assert parser.validate(parser.arf_schemas_path) == result
+
+
+@pytest.mark.parametrize("file_path, number_of_cpe_platforms", [
+    (PATH_TO_ARF, 13),
```

Comment:
In general, these are some huge XML files. But, for unit tests, I would expect to have many minimal ARF or XCCDF files to have the testing more focused.